astropy/utils/parsing.py

```diff
@@ -37,7 +47,7 @@ def _add_tab_header(filename, package):
 
 
 @contextlib.contextmanager
-def _patch_get_caller_module_dict(module):
+def _patch_get_caller_module_dict(module: ModuleType) -> Generator[None, None, None]:
```

Comment:
Can't wait for Python 3.13 (https://docs.python.org/3/library/typing.html#typing.Generator)
> Changed in version 3.13: Default values for the send and return types were added.

Reply:
Agreed.